docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+version: "3"
+services:
+  scraper:
+    image: asherrill/buycott_scraper:${TAG}
+    environment:
+      - RABBITMQ_USER=${RABBITMQ_USER}
+      - RABBITMQ_USER_PW=${RABBITMQ_USER_PW}
+      - RABBITMQ_DEV_HOST=${RABBITMQ_DEV_HOST}
+      - RABBITMQ_PORT=${RABBITMQ_PORT}
+      - MONGODB_DEV_URI=${MONGODB_DEV_URI}
+    container_name: buycott_scraper
+    command: "python -u receive_barcode.py" # The flag -u give unbuffered output, in order to see logs from python script
+    restart: unless-stopped
+  watchtower:
+    image: containrrr/watchtower
+    container_name: watchtower
+    environment:
+      - WATCHTOWER_CLEANUP="true"
+      - TZ=Europe/Berlin
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+      - /root/.docker/config.json:/config.json
+    command: --interval 30 --debug --cleanup
```

Comment:
Here u have the watchtower i see 👍 
I would lower the interval otherwise you wont be able to read the logs.
Maybe every 5 min or even slower

Reply:
Jap, I totally with u. What do you think... maybe even 15 min?

Reply:
Yeah we can even put hours :D

Reply:
I think in the reflection essay i wrote taht we only check once per day at 3 am to keep the downtime down